utils/utils.go

```diff
@@ -88,3 +92,32 @@ func CreateKubeClient(scheme *runtime.Scheme, kubeconfig string) (runtimeclient.
 	return runtimeclient.New(config, runtimeclient.Options{Scheme: scheme,
 		WarningHandler: runtimeclient.WarningHandlerOptions{SuppressWarnings: true}})
 }
+
+func RunOnce(name string, log *logrus.Logger, f interface{}, args ...interface{}) error {
+	doneFile := path.Join(common.BackupChecksDir, name+".done")
```

Comment:
I think better to provide dir, this will allow testing

Reply:
ack, changed!